go/cmd/vtctldclient/command/vreplication/common/utils.go

```diff
@@ -192,6 +212,7 @@ func AddCommonFlags(cmd *cobra.Command) {
 
 func AddCommonCreateFlags(cmd *cobra.Command) {
 	cmd.Flags().StringSliceVarP(&CreateOptions.Cells, "cells", "c", nil, "Cells and/or CellAliases to copy table data from.")
+	cmd.Flags().BoolVar(&CreateOptions.AllCells, "all-cells", false, "Copy table data from any existing cell.")
```

Comment:
this _might_ conflict with a subcommand's option (i haven't checked) but a shortopt `-a` (to match `--cells|-c`) would be nice here.

Reply:
Good idea. We're good:
```
go/cmd/vtctldclient/command/vreplication/common/utils.go:	cmd.Flags().BoolVarP(&CreateOptions.AllCells, "all-cells", "a", false, "Copy table data from any existing cell.")
go/cmd/vtctldclient/command/vreplication/workflow/workflow.go:	getWorkflows.Flags().BoolVarP(&getWorkflowsOptions.ShowAll, "show-all", "a", false, "Show all workflows instead of just active workflows.")
```